ibm/flex/structures.go

```diff
@@ -2558,11 +2559,44 @@ func UpdateGlobalTagsUsingCRN(oldList, newList interface{}, meta interface{}, re
 		if err != nil {
 			return fmt.Errorf("[ERROR] Error updating database tags %v : %s\n%s", add, err, resp)
 		}
+		response, errored := waitForTagsAvailable(meta, resourceID, resourceType, tagType, news, 30*time.Second)
```

Comment:
timeout is set to 30 seconds rather than 1 minute, each repoll is set at 3 seconds